layouts/partials/announcement.html

```diff
@@ -24,6 +24,9 @@ <h4>
         </h4>
         {{ end }}
         <p>{{ .message | markdownify }}</p>
+        {{- if eq .dismissParameters.dismissible true -}}
+        <button  id="banner-dismiss" class="button" style="display: none;" data-ttl={{ .dismiss_params.showDismissedAnnouncementAfterDays }}>{{ T "banner_acknowledgement" }}</button> <!-- Only JS enabled users will see this button -->
```

Comment:
nit: I might use a `printf` to ensure that `.dismiss_params.showDismissedAnnouncementAfterDays` is a string representation of a integer.